pkgs/development/tools/build-managers/meson/setup-hook.sh

```diff
@@ -31,7 +31,69 @@ mesonConfigurePhase() {
     runHook postConfigure
 }
 
+mesonBuildPhase() {
```

Comment:
I would skip this one, you don't need it.

Reply:
Yeah I mostly added this because I was worried it would be more confusing if meson was called directly for some tasks but not others.  I'd be happy to drop it too.  Whatever @jtojnar would prefer as the package maintainer.

Reply:
I like the "less code is better" approach :) and it means you don't have to sync ninja handling over from elsewhere...